src/material-experimental/popover-edit/popover-edit.spec.ts

```diff
@@ -564,6 +564,10 @@ matPopoverEditTabOut`, fakeAsync(() => {
       });
 
       describe('edit lens', () => {
+        function expectPixelsToEqual(actual: number, expected: number) {
+          expect(Math.round(actual)).toBe(Math.round(expected));
```

Comment:
Usually I do `Math.floor` or `Math.ceil` for assertions like this, because with round you could get a mismatch for `1.4` vs `1.6`.

Reply:
Yeah, that is tricky. I checked and saw lots of `Math.round` assertions in our code base. We don't seem to do it consistently. One benefit of round will be that we potentially spot one-off issues easier (as the browser potentially will round the pixels depending). I'll switch it to `Math.floor`.

Reply:
done